src/cascadia/TerminalCore/TerminalApi.cpp

```diff
@@ -389,8 +389,9 @@ bool Terminal::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle) noex
 
     switch (cursorStyle)
     {
-    case DispatchTypes::CursorStyle::BlinkingBlockDefault:
-        [[fallthrough]];
+    case DispatchTypes::CursorStyle::UserDefault:
+        finalCursorType = _defaultCursorShape;

```

Comment:
Is there a way to know whether blinking should be allowed, without `switch` on `_defaultCursorShape`?

Reply:
In Windows Terminal I think the default for _shouldBlink_ would always be true (at least for now), since there is no option to change that as far as I know (I think that's issue #1379).

Reply:
Huh. So you can turn off the blinking using VT escape codes. But you can not do that using plain settings. That's just...bizarre,

Update: I almost forgot that I had a PR #2892 which was trying to fix this. How time flies.